thorn/include/kernel/mailbox.h

```diff
@@ -32,9 +32,12 @@ typedef enum {
 typedef struct {
     unsigned int data    : 28;// most significant bits contain shared memory address
```

Comment:
```suggestion
    unsigned int data    : 28;// most significant bits (MSB) contain 28 MSB of shared memory address so that `mailbox_message_t` is in the proper format the mailbox expects. The 4 LSB are 0 anyways.
```
see my explanation for `mailbox_message_to_register_value`